addons/io_scene_gltf2/blender/exp/gltf2_blender_gather_image.py

```diff
@@ -27,14 +27,16 @@
 from io_scene_gltf2.io.exp.gltf2_io_user_extensions import export_user_extensions
 
 
-@cached
+# @cached
 def gather_image(
         blender_shader_sockets: typing.Tuple[bpy.types.NodeSocket],
-        export_settings):
+        export_settings,
+        export_image_class = ExportImage
```

Comment:
The logic here is that all of the extensions related to image formats right now actually start at the `texture` level, so its likely they will be calling `gather_image` and can provide an override. Its possible this could instead be handled in reverse via something like a `pre_gather_image_hook` that could return some data, but it would likely require duplicating some functionality.